typescript/infra/scripts/utils.ts

```diff
@@ -196,6 +204,43 @@ export async function getMultiProviderForRole(
   return multiProvider;
 }
 
+export async function getKeysForRole(
```

Comment:
May be worth noting this relates to private keys and therefore only works for key types that allow private keys being extracted. Same for `getAddressesForKey`